packages/ns-api/files/ns.update

```diff
@@ -86,11 +87,15 @@ def install_package_updates():
     return {"result": "success"}
 
 def check_system_update():
+    e_uci = EUci()
     current_version = get_system_version()
     ret = {"currentVersion": f'NethSecurity {current_version}', "lastVersion": "", "scheduledAt": get_update_schedule()}
     try:
-        channel = get_distfeed_channel()
-        with urllib.request.urlopen(f"https://updates.nethsecurity.nethserver.org/{channel}/latest_release") as resp:
+        url = f"{e_uci.get('ns-plug', 'config', 'repository_url')}"
```

Comment:
No need to cast to string:
```suggestion
        url = e_uci.get('ns-plug', 'config', 'repository_url', default="https://updates.nethsecurity.nethserver.org")
```

Reply:
I'd remove the default, it is set anyway, by default pyuci returns a string type
```suggestion
        url = e_uci.get('ns-plug', 'config', 'repository_url')
```

Reply:
ok remove the default and the f-string